docs/arcadia-users-group/20240730-arcadiathemer/20240730-arcadiathemer.Rmd

```diff
@@ -0,0 +1,143 @@
+```{r, indices}
+ggplot(mtcars, aes(x = hp, fill = as.factor(cyl))) +
+  geom_density(alpha = 0.8, linewidth = 0) + # remove border line from filled-in density plots
+  theme_arcadia() +
+  scale_fill_arcadia(palette_name = "blue_shades", start=2, end=5) +
+  scale_y_continuous(expand=c(0,0)) +
+  scale_x_continuous(expand=c(0,0)) # remove whitespace between both axes and the plot
+```
+The `scale` functions are used to specify which palettes to use. The `gradient` functions are used to access the gradient palettes, which work in the same way as the `scale` functions using `color` or `fill`:
+```{r, gradient}
+ggplot(data = mtcars, aes(x = hp, y = mpg, color = hp)) +
+ geom_point(size=2.5) + 
+ theme_arcadia() + 
+ gradient_color_arcadia(palette_name = "lisafrank")
+```
+There are also bicolor gradients available that are useful for heatmap plots. You can also remove the background color with `background=FALSE`, which is recommended when exporting plots, which is described below. 
+```{r, heatmap}
+library(reshape2)
+
+# heatmap of correlation matrix
+data(iris)
+iris_data <- iris[, 1:4]
+cor_matrix <- cor(iris_data)
+melted_cor_matrix <- (melt(cor_matrix))
+
+ggplot(melted_cor_matrix, aes(x=Var1, y=Var2, fill=value)) +
+  geom_tile() +
+  theme_arcadia(x_axis_type = "categorical", y_axis_type = "categorical", background = FALSE) +
+  gradient_fill_arcadia(palette_name = "reds") + 
+  theme(axis.text.x = element_text(angle = 45, hjust = 1),
+        legend.position = "top", axis.line = element_blank()) +
+  labs(x = "", y = "") +
+  scale_y_discrete(expand=c(0,0)) +
+  scale_x_discrete(expand = c(0,0))
+```
+
+## Accessing palettes and specific colors
+To view all the color palette options and the individual hex codes that comprise each palette, you can view these with `show_arcadia_palettes()` or `show_arcadia_gradients()`. The gradients also show the positions for each individual color in the gradient. You can use these lists of hex codes to create custom or reordered sets of palettes.
+```{r, show_colors}
+show_arcadia_palettes()
+show_arcadia_gradients()
+```
+
+## Exporting plots
+To save plots, we have a custom `save_arcadia_plot()` function built on top of `ggsave()` that helps you export plots that adhere to our size guidelines and can be used with the Illustrator templates. The different plot size options are `"full_wide", "float_wide", "half_square", "full_square"`, or `"float_square"`. These panel sizes adhere to the panel sizes available in the Illustrator templates. 
+
+If you want to save plots in different formats (such as PNG or JPG) or sizes that aren't available through this convenience function, you can still use the regular `ggsave()` function. Additionally for the background to be transparent in exported plots you need to set this argument to `FALSE` in the `theme_arcadia()` function:
```

Comment:
Nit: explicitly referring to the argument might make it easier for readers to figure out what to look at in the code block below

```suggestion
If you want to save plots in different formats (such as PNG or JPG) or sizes that aren't available through this convenience function, you can still use the regular `ggsave()` function. Additionally for the background to be transparent in exported plots you need to set the `background` argument to `FALSE` in the `theme_arcadia()` function:
```

Reply:
fixed